autoinstall/Ubuntu/Flutter/user-data.j2

```diff
@@ -0,0 +1,54 @@
+#cloud-config
+autoinstall:
+  version: 1
+  early-commands:
+    - echo '{{ autoinstall_start_msg }}' >/dev/ttyS0
+  locale: en_US.UTF-8
+  keyboard:
+    layout: us
+  network:
+    version: 2
+    ethernets:
+      all-eth:
+        match:
+          name: "ens*"
+        dhcp4: yes
+  storage:
+    layout:
+      name: direct
+    grub:
+      reorder_uefi: False
+  timezone: US/Eastern
+  identity:
+    hostname: ubuntu
+    realname: ubuntu
+    username: {{ vm_username }}
+    password: {{ vm_password_hash }}
+  user-data:
+    users:
+      - name: root
+        lock_passwd: false
+        hashed_passwd: {{ vm_password_hash }}
+        ssh_authorized_keys:
+          - {{ ssh_public_key }}
+{% if new_user is defined and new_user != 'root' %}
+      - name: {{ new_user }}
+        lock_passwd: false
+        hashed_passwd: {{ vm_password_hash }}
+        sudo: ALL=(ALL) NOPASSWD:ALL
+        ssh_authorized_keys:
+          - {{ ssh_public_key }}
+{% endif %}
+  apt:
+    preserve_sources_list: false
+    geoip: true
+  late-commands:
+    - echo "# apt-get update" >> /dev/ttyS0
+    - curtin in-target --target=/target -- apt-get update -y >> /dev/ttyS0
+    - echo "# apt-get install build-essential openssh-server open-vm-tools open-vm-tools-desktop cloud-init rdma-core rdmacm-utils ibverbs-utils" >> /dev/ttyS0
+    - curtin in-target --target=/target -- apt-get install -y --force-yes build-essential openssh-server open-vm-tools open-vm-tools-desktop cloud-init rdma-core rdmacm-utils ibverbs-utils >> /dev/ttyS0
```

Comment:
Can these packages be put under `packages:` section like live-server?

Reply:
From the test result:
1) Sometimes package is failed to install which stuck the autoinstall when these packages are put to packages section. 
2) We can put the package installing message to serial_output.log when these packages are put to late-command section. 


Best regards
Yuhua Zou